src/App/MeasureManager.h

```diff
@@ -25,18 +25,21 @@
 
 
 #include <vector>
-#include <tuple>
 #include <functional>
 #include <string>
 #include <Python.h>
```

Comment:
It should be tried to remove Python.h because this includes 60+ further header files. PyObject can be forward declared as: `using PyObject = struct _object;`